src/Command/ClearCommand.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+/*
+ * This file is part of the FOSHttpCacheBundle package.
+ *
+ * (c) FriendsOfSymfony <http://friendsofsymfony.github.com/>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace FOS\HttpCacheBundle\Command;
+
+use FOS\HttpCacheBundle\CacheManager;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+/**
+ * A command to trigger cache invalidation by path from the command line.
+ *
+ * @author Alexander Schranz <alexander@sulu.io>
+ */
+class ClearCommand extends BaseInvalidateCommand
+{
+    use PathSanityCheck;
+
+    protected static $defaultName = 'fos:httpcache:invalidate';
```

Comment:
i would prefer to call it fos:httpcache:clear to be explicit that this does not take options...